libs/ui/src/hooks/use_devices.test.ts

```diff
@@ -365,59 +365,63 @@ test('can handle logs for a random device as expected', async () => {
   );
 
   // Prevent `act` warning.
-  await waitForNextUpdate();
+  unmount();
 });
 
 test('periodically polls for computer battery status', async () => {
   jest.useFakeTimers();
   const hardware = new MemoryHardware();
   const logger = fakeLogger();
-  const { result, waitForNextUpdate } = renderHook(() =>
+  const { result, unmount } = renderHook(() =>
     useDevices({ hardware, logger })
   );
   expect(result.current).toEqual(emptyDevices);
 
-  await waitForNextUpdate();
```

Comment:
Does `waitForNextUpdate` not work anymore?

Reply:
It doesn't exist on the `renderHook` API added into mainline RTL. In the unmount case, I think we're supposed to use `unmount`. In the waiting on a status case, I think we're just supposed to use `waitFor` as is the pattern elsewhere.

Reply:
Weird that they got rid of that. Oh well